thejmdw/gamer-rater-server
# Thejmdw book3 chapter2

To Test:

in project directory run
`python3 manage.py test tests -v 1 `

8 tests should run and pass.
